public/index.html

```diff
@@ -0,0 +1,370 @@
+          <figcaption>
+            <h4>Hexagon onepage</h4>
+            <p>Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-3">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/sketches.jpg" alt="Pencil sketches"></a>
+          <figcaption>
+            <h4>Sketches icon</h4>
+            <p>Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-4">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/logotypes.jpg" alt="Logos screenshot"></a>
+          <figcaption>
+            <h4>Logotypes volume</h4>
+            <p>UI design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-5">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/flight.jpg" alt="Flight screenshot"></a>
+          <figcaption>
+            <h4>Flight control</h4>
+            <p>Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-6">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/Ytube.jpg" alt="Iphone app screenshot"></a>
+          <figcaption>
+            <h4>Youtube app</h4>
+            <p>Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-7">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/management.jpg" alt="Management screenshot"></a>
+          <figcaption>
+            <h4>Management</h4>
+            <p>UI design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-8">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/koreanbeef.jpg" alt="Menu screenshot"></a>
+          <figcaption>
+            <h4>Korean beef</h4>
+            <p>Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-9">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/thumbs.jpg" alt="Like dislike screenshot"></a>
+          <figcaption>
+            <h4>Facebook thumbs</h4>
+            <p>Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+      </section>
+      
+    </article>
+    <span class="white-notch"></span>
+    <!-- about  -->
+    <article class="about">
+      <h1>ABOUT</h1>
+      <section class="flex-container flex-paragraphs">
```

Comment:
Here, you do have multiple thematic chunks, but I think that's a stretch. I'd rather see `div`s here. Additionally, a `section` should contain a heading element (`h1-h6`).